src/V3Error.h

```diff
@@ -96,6 +96,7 @@ class V3ErrorCode final {
         IGNOREDRETURN,  // Ignoring return value (function as task)
         IMPERFECTSCH,   // Imperfect schedule (disabled by default). Historical, never issued.
         IMPLICIT,       // Implicit wire
+        IMPLSTATICFUNC, // Implicit static function
```

Comment:
It also applies to a task, so I suggest IMPLICITSTATIC.

Reply:
Done